examples/stable_diffusion_xl/gm/modules/embedders/open_clip/tokenizer.py

```diff
@@ -161,6 +161,54 @@ def decode(output_ids: Tensor):
     return _tokenizer.decode(output_ids)
 
 
+def lpw_tokenize(
+    texts: Union[str, List[str]],
+    context_length: int = 77,
+    max_embeddings_multiples: int = 4,
```

Comment:
suggestion: allow to set `max_embeddings_multiples` via yaml config or arg parser